staging/src/k8s.io/apimachinery/pkg/util/net/http.go

```diff
@@ -443,3 +444,19 @@ func CloneHeader(in http.Header) http.Header {
 	}
 	return out
 }
+
+// NewDefaultTransport returns a new instance of http.DefaultTransport
+func NewDefaultTransport() *http.Transport {
+	return &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
```

Comment:
this loses `NewProxierWithNoProxyCIDR(http.ProxyFromEnvironment)` capabilities, which means apiservers running with HTTP_PROXY can't punch holes based on a pod or service cidr

Reply:
Linking to https://github.com/kubernetes/kubernetes/pull/76264/files#r273378950